src/test/java/io/dropwizard/bundles/assets/AssetServletTest.java

```diff
@@ -132,8 +144,9 @@ public void setup() throws Exception {
     servletTester.addServlet(NoCharsetAssetServlet.class, NOCHARSET_SERVLET + '*');
     servletTester.addServlet(RootAssetServlet.class, ROOT_SERVLET + '*');
     servletTester.addServlet(MimeMappingsServlet.class, MIME_SERVLET + '*');
+    servletTester.addServlet(CachingServlet.class, CACHE_SERVLET + '*');
 
-    ServletHolder servlet = new ServletHolder(new MultipleMappingsServlet());
+    ServletHolder servlet = new ServletHolder(MultipleMappingsServlet.class);
```

Comment:
Tweaking this removed and IntelliJ warning about a constructor that could be made package private.  Normally I'd just make the constructor package private, but that would be inconsistent with all of our other AssetServlet subclasses in this test case.